regression/scripts/weights/eVTOL_Weights_Buildup_Regression.py

```diff
@@ -0,0 +1,352 @@
+# eVTOL_Weights_Buildup_Regression.py
+#
+# Author: Jordan Smart, Feb. 2018
```

Comment:
"Created"

Reply:
Changed throughout project to comply with style guide.